keyboards/contra/keymaps/bramver/keymap.c

```diff
@@ -0,0 +1,130 @@
+/* Copyright 2018 darm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum emoji_map {
+  UNAM, // unamused 😒
+  HEYE, // smiling face with heart shaped eyes 😍
+  OK, // ok hand sign 👌
+  SMIR, // smirk 😏
+  PRAY, // pray 🙏
+  CELE, // celebration 🙌
+  COOL, // smile with sunglasses 😎
+  EYES, // eyes
+  THNK, // BIG THONK
+  NAIL, // Nailcare
+  SOS, // Vuile sos
+  REDB, // Red B
+  HNDR, // 100
+  MONY,
+  FIRE,
+  CAR,
+  BUTT,
+  BNIS,
+  CUM,
+  CLAP,
+  TRIU, // Fart from nose
+  SCRM,
+  VOMI,
+  DTIV, // Detective
+  EXPL, // Brainsplosion
+  HAIR, // Haircut
+  DANC, // Salsa dancer
+  STRN, // Stronk
+  LEFT, // Point Left
+  RGHT, // Point Right
+};
+
+const uint32_t PROGMEM unicode_map[] = {
+  [UNAM] = 0x1F612,
+  [HEYE] = 0x1f60d,
+  [OK]   = 0x1F44C,
+  [SMIR] = 0x1F60F,
+  [PRAY] = 0x1F64F,
+  [CELE] = 0x1F64C,
+  [COOL] = 0x1F60E,
+  [EYES] = 0x1F440,
+  [THNK] = 0x1F914,
+  [NAIL] = 0x1F485,
+  [SOS] = 0x1F198,
+  [REDB] = 0x1F171,
+  [HNDR] = 0x1F4AF,
+  [MONY] = 0x1F480,
+  [FIRE] = 0x1F525,
+  [CAR] = 0x1F697,
+  [BUTT] = 0x1F351,
+  [BNIS] = 0x1F346,
+  [CUM] = 0x1F4A6,
+  [CLAP] = 0x1F44F,
+  [TRIU] = 0x1F624,
+  [SCRM] = 0x1F631,
+  [VOMI] = 0x1F92E,
+  [DTIV] = 0x1F575,
+  [EXPL] = 0x1F92F,
+  [HAIR] = 0x2640,
+  [DANC] = 0x1F483,
+  [STRN] = 0x1F4AA,
+  [LEFT] = 0x1F448,
+  [RGHT] = 0x1F449,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
7 space KC_TRNS and 7 X KC_NO are already included in `QMK_KEYBOARD_H`. You won't need to add this here.

Reply:
```suggestion
```

Reply:
Ah didn't know that, will do!